astropy/cosmology/tests/test_parameter.py

```diff
@@ -139,15 +154,21 @@ def test_Parameter_derived(self, cosmo_cls, all_parameter):
             all_parameter.name not in cosmo_cls.__parameters__
         )
 
+    def test_Parameter_default(self, cosmo_cls, all_parameter):
+        """Test :attr:`astropy.cosmology.Parameter.default`."""
+        assert hasattr(all_parameter, "default")
+        assert isinstance(
```

Comment:
I think I'd write this as
```
assert all_parameter.default is MISSING or all_parameter.default is None or isinstance(all_parameter.default, (float, u.Quantity))
```
This tests something closer to what is actually done (and `Sentinel` is an implementation detail).

Reply:
I like separating out the MISSING, but NoneType is py3.10+, so there's no need to separate that out. Ruff will auto-upgrade `type(None)` -> `NoneType` when we next bump versions.